tests/test_git.py

```diff
@@ -417,6 +422,49 @@ def test_fetch_refspecs(
     assert baz_rev == scm.get_ref("refs/foo/baz")
 
 
+@pytest.mark.skip_git_backend("pygit2", "gitpython")
+@pytest.mark.parametrize("use_url", [True, False])
+@pytest.mark.xfail(
+    os.name == "nt",
+    reason="file:// urls are not supported by dulwich on windows",
```

Comment:
This xfail can be removed when https://github.com/jelmer/dulwich/pull/965 is merged and dulwich is upgraded

Reply:
looks like this should be fixed in the 0.20.38 which was just released?

Reply:
Indeed, let me get rid of this, but before that, we need to get rid of the `file://` urls workaround for windows

Reply:
https://github.com/iterative/scmrepo/pull/68